licenses/models.py

```diff
@@ -139,6 +139,84 @@ def save(self, *args, **kwargs):
         )
         super().save(*args, **kwargs)
 
+    def _get_save_path(self):
+        """
+        If saving the deed or license as a static file, this returns
+        the relative path where the saved file should be, not including
+        the actual filename.
+        """
+        license = self.license
+        firstdir = (
+            "publicdomain" if license.license_code.lower() == "cc0" else "licenses"
+        )
+        if license.version == "3.0":
+            return os.path.join(
+                firstdir, license.version, license.jurisdiction_code or "xu"
```

Comment:
`    For jurisdiction, I used “xu” to mean “unported”.` >> can we add this comment here as well.

Reply:
updated